### PR TITLE
Feature/issue 17

### PR DESCRIPTION
**Pull request by lariali1986**
token expiration+image swiper+spinner+multi selection of cards

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -21,10 +21,16 @@
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
         "backgroundColor": "#ffffff"
-      }
+      },
+      "package": "com.alilari1986.CirclesApp"
```
Comment (sharabiania) [inline app.json:25]: remove `alilari1986` from the package name

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "circlesapp",
+  "name": "circlesapp1",
```
Comment (sharabiania) [inline package.json:2]: name: CirclesApp

File: util/auth.js [hunk 3]
```diff
@@ -1,51 +1,34 @@
...
 }
+
+
+
+
+
+
+
+
+
```
Comment (sharabiania) [inline util/auth.js:34]:
```suggestion
```

File: app.json (hunk 1 above)
Comment (lariali1986) [reply app.json:25]: Done.

File: util/auth.js (hunk 3 above)
Comment (lariali1986) [reply util/auth.js:34]: Done.

**Review by sharabiania — APPROVED** [review]

**Pull request merged**